Test/AwsCommonRuntimeKitTests/CrtXCBaseTestCase.swift

```diff
@@ -3,19 +3,23 @@
 
 import XCTest
 import AwsCommonRuntimeKit
+import AwsCCommon
 
 class CrtXCBaseTestCase: XCTestCase {
     internal let allocator = TracingAllocator(tracingStacksOf: defaultAllocator)
     var logging: Logger?
 
     override func setUp() {
         super.setUp()
-        AwsCommonRuntimeKit.initialize(allocator: self.allocator)
         logging = Logger(pipe: stdout, level: .trace, allocator: defaultAllocator)
+
+        AwsCommonRuntimeKit.initialize(allocator: self.allocator)
     }
 
     override func tearDown() {
+        aws_thread_join_all_managed()
```

Comment:
what does this do?

Reply:
It shuts down all CRT threads. I thought a thread might be hanging out from the ELG, but I was wrong. This didn't help.